### ISSUE TITLE
Device Connection issue!

### ISSUE THREAD

**Issue by vt57299** [pollen-robotics/pollen-vision#140]
To run certain samples, a device connection is required. Can we use the Reachy simulator, or a physical Reachy robot, and a camera needed?

**Comment by apirrone** [comment]
Hello @vt57299 

You can run the models with any image input, for example, as shown in our readme : 

```python
import cv2

from pollen_vision.vision_models.object_detection import OwlVitWrapper
from pollen_vision.vision_models.object_segmentation import MobileSamWrapper
from pollen_vision.perception.utils import Annotator, get_bboxes


owl = OwlVitWrapper()
sam = MobileSamWrapper()
annotator = Annotator()

cap = cv2.VideoCapture(0) # This uses your webcam

while True:
    ret, frame = cap.read() 
    predictions = owl.infer(
        frame, ["paper cups"]
    )  # zero-shot object detection | put your classes here
    bboxes = get_bboxes(predictions)

    masks = sam.infer(frame, bboxes=bboxes)  # zero-shot object segmentation
    annotated_frame = annotator.annotate(frame, predictions, masks=masks)

    cv2.imshow("frame", annotated_frame)
    if cv2.waitKey(1) & 0xFF == ord("q"):
        cv2.destroyAllWindows()
        break

```
       
You just have to get the `frame` any way you want. In this example, it uses your webcam, but you can also load an image, a video or any camera, you only have to be able to fetch frames. 

You can also get the frames using the Reachy SDK (https://github.com/pollen-robotics/reachy-sdk) connecting to a simulated robot, or a physical one. Here you can see how to get the images from Reachy's cameras : https://docs.pollen-robotics.com/sdk/first-moves/cameras/

**Comment by vt57299** [comment]
This one I am able to run just fine, but I am asking about the other examples like "grasp_demo.py", "live_pcl.py".

**Comment by apirrone** [comment]
These examples are not really documented. `live_pcl.py` is just a script to display a point cloud from rgb + depth data using open3D. To run it, you either have to have the same cameras as us (Luxonis OAK-D-SR), or implement a `CameraWrapper` with your own rgbd camera (https://github.com/pollen-robotics/pollen-vision/blob/develop/pollen_vision/pollen_vision/camera_wrappers/camera_wrapper.py). 

`grasp_demo.py` can give you an example on how to use rgbd cameras with pollen vision to grasp objects with a robot, but it was more of an internal script, you would no be able to run it directly unless you have a physical Reachy robot with the camera we used, mounted the same way we did for this demo.
